Source/JavaScriptCore/b3/air/AirAllocateRegistersByGraphColoring.cpp

```diff
@@ -1739,7 +1758,7 @@ class ColoringAllocator : public AllocatorType<IndexType, InterferenceSet, bank>
                 return false;
 
             if (tmpWidth->defWidth(inst.args[0].tmp()) > Width32
-                && tmpWidth->useWidth(inst.args[1].tmp()) > Width32)
+                || tmpWidth->useWidth(inst.args[1].tmp()) > Width32) // OOPS fixme
```

Comment:
OOPS is not supported format in WebKit. Always use FIXME.

Reply:
Yeah, I wanted to make sure this wouldn't get landed. Is it correct that the merge-queue won't allow landing OOPS?